GUI/QtQuick/qml/WalletProperties/ExportWOWallet.qml

```diff
@@ -169,8 +170,9 @@ ColumnLayout  {
     CustomButton {
         id: confirm_but
 
-        enabled: bsApp.settingExportDir.length !== 0
+        //enabled: bsApp.settingExportDir.length !== 0
```

Comment:
Why was this commented out? Shouldn't we set the export dir first?

Reply:
Default value is documents directory.
It never be empty.
But